### PR TITLE
Adding myself as a student and added a project

### PR DESCRIPTION
**Pull request by muhammednagy**
**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- i added myself as a student and added my blog

### CODE REVIEW

**Comment by Elgayar** [comment]
@muhammednagy Hey, please add a live link or a screenshot and write a description for your pull request!

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
please resize your image to 240x240

File: _data/blogs.yml [hunk 1]
```diff
@@ -183,7 +183,7 @@
   blog_img: adarsh-blog3.png
   article: My experience with Fossasia
   article_link: https://idealadarsh.wordpress.com/2016/12/12/my-experience-with-fossasia/
-
+
```
Comment (abishekvashok) [inline _data/blogs.yml]: Yikes there is a trailing white space please remove it

**Review by abishekvashok — CHANGES_REQUESTED** [review]
**To resolve merge conflicts follow the tutorial here:  https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/**

And remember github provides you to host your repository on it, so if you are on the gh-pages or master branch you can deploy your site online with github, since you are on the gh-pages you are ready to go.
**So to see your repository online you have to go to [live link]:** `https://username.github.io/repository_name/`
**In your case it for this repository (meaning your fork) it should be [live link]:** https://muhammednagy.github.io/gci16.fossasia.org/

Then generate the screenshot, on Linux and Windows this is achieved by pressing `Ctrl + PrtScn`

**You should add the description where the PR body states it to specify and not to the top please do follow our coding practices for good results**.

_Note: You can change the SSL on and off for it. Just go to settings to do so!_

happy coding :)

File: _data/blogs.yml (hunk 1 above)
Comment (robbyoconnor) [reply _data/blogs.yml]: coala will take care if it

File: _data/blogs.yml [hunk 2]
```diff
@@ -207,3 +207,7 @@
   article: My Participation in Google Code-in
   article_link: https://prajwalpatil.tech.blog/2016/12/27/experience-in-google-code-in/
 
+- name: Muhammed Nagy
+  blog: https://blog.muhnagy.com
+  article: About me
+  artivle_link: https://blog.muhnagy.com
```
Comment (robbyoconnor) [inline _data/blogs.yml]: Typo here.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

**Comment by robbyoconnor** [comment]
@muhammednagy, we fixed a lot of the issues. You did not actually read those checkboxes. it said 240x240. I unchecked it. Please check it when you actually completed the steps.

File: _data/blogs.yml (hunk 1 above)
Comment (Abhi2424shek) [inline _data/blogs.yml]: I take it back sorry.

**Comment by robbyoconnor** [comment]
@muhammednagy I got charitable and fixed all issues. Next time, be sure you read things **_closely_**. That said, please submit your task so I can approve it.

**Comment by robbyoconnor** [comment]
I fixed it.

**Comment by muhammednagy** [comment]
Thanks 
i had exams and i was busy. i was unable to fix my commit or see this comments 
i won't do it again 
thanks again @robbyoconnor

**Comment by abishekvashok** [comment]
No probs we would be more happy to see some of your PRs merged @muhammednagy

**Pull request closed without merging**